test/functional/feature_loan_estimateloan.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2019 The Bitcoin Core developers
+# Copyright (c) DeFi Blockchain Developers
+# Distributed under the MIT software license, see the accompanying
+# file LICENSE or http://www.opensource.org/licenses/mit-license.php.
+"""Test Loan Scheme."""
+
+from test_framework.test_framework import DefiTestFramework
+
+from test_framework.authproxy import JSONRPCException
+from test_framework.util import assert_equal
+import time
+
+class DepositToVaultTest (DefiTestFramework):
```

Comment:
Very minor. Rename test from DepositToVaultTest, also update the comment "Test Loan Scheme" above which is also not updated on the DepositToVaultTest used as a template.